flang/include/flang/Optimizer/Builder/Runtime/Reduction.h

```diff
@@ -18,33 +25,33 @@ class FirOpBuilder;
 
 namespace fir::runtime {
 
-/// Generate call to all runtime routine.
-/// This calls the descriptor based runtime call implementation of the all
+/// Generate call to `all` runtime routine.
+/// This calls the descriptor based runtime call implementation of the `all`
 /// intrinsic.
 void genAllDescriptor(fir::FirOpBuilder &builder, mlir::Location loc,
                       mlir::Value resultBox, mlir::Value maskBox,
                       mlir::Value dim);
 
-/// Generate call to any runtime routine.
-/// This calls the descriptor based runtime call implementation of the any
+/// Generate call to `any` runtime routine.
+/// This calls the descriptor based runtime call implementation of the `any`
 /// intrinsic.
 void genAnyDescriptor(fir::FirOpBuilder &builder, mlir::Location loc,
                       mlir::Value resultBox, mlir::Value maskBox,
                       mlir::Value dim);
 
-/// Generate call to all runtime routine. This version of all is specialized
+/// Generate call to `all` runtime routine. This version of `all` is specialized
 /// for rank 1 mask arguments.
 /// This calls the version that returns a scalar logical value.
 mlir::Value genAll(fir::FirOpBuilder &builder, mlir::Location loc,
                    mlir::Value maskBox, mlir::Value dim);
 
-/// Generate call to any runtime routine. This version of any is specialized
+/// Generate call to `any` runtime routine. This version of `any` is specialized
 /// for rank 1 mask arguments.
 /// This calls the version that returns a scalar logical value.
 mlir::Value genAny(fir::FirOpBuilder &builder, mlir::Location loc,
                    mlir::Value maskBox, mlir::Value dim);
 
-/// Generate call to Count runtime routine. This routine is a specialized
+/// Generate call to `count` runtime routine. This routine is a specialized
```

Comment:
These comment changes are strange. The runtime routine is, in fact, named "Count" (with a name mangling macro wrapper). Generally, flang uses all caps names to refer to Fortran artifacts, so "COUNT" would be the typical way of referring to the intrinsic in, say, a presentation slide.

It's arguable that `count` isn't just wrong even. The backquotes suggest it is an identifier in the code, such as an argument name. It clearly is not.

Reply:
Well it was ask be reviewers. I'm personally fine reverting to the original routine name.